controllers/dataprotection/backup_controller.go

```diff
@@ -854,6 +856,42 @@ func setConnectionPasswordAnnotation(request *dpbackup.Request) error {
 	return nil
 }
 
+func setEncryptedSystemAccountsAnnotation(request *dpbackup.Request, cluster *appsv1alpha1.Cluster) error {
+	var (
+		usernameKey = "username"
```

Comment:
Hi @shanshanying, could you confirm that these hard-coded keys are safe to use for the abstraction of the system account in KubeBlocks? Are they the same for all kinds of DB engines?

Reply:
I modify this to `usernameKey := constant.AccountNameForSecret
	passwordKey := constant.AccountPasswdForSecret`.
Use the const keys defined in pkg/constant/const.go, which are used to build system account secrets while component reconciling.